helper/user_comms_helper.rb

```diff
@@ -1,13 +1,13 @@
-class UserComms
+class UserCommsHelper
 
   ERROR_INITIALISE_WITH_STRING_IO = "Initialise with StringIO objects"
   TELL_USER_NO_DEVELOP_TAGS = "No develop tags exist for this repository"
   ASK_USER_INCREMENT_TYPE = "Would you like to do a major, minor, or patch increment?"
   ERROR_SELECT_ACCEPTED_INCREMENT_TYPE = "Error: please select major, minor or patch update"
   ERROR_NEXT_TAG_NOT_ASSIGNED = "Next tag has not been assigned"
   ERROR_SELECT_Y_OR_N = "Error: please select y/n"
-  ERROR_NO_ENVIRON_SELECTED = "Error: please select to apply develop, test, or stage tag\
-                              \n e.g. ruby deploy.rb develop\n"
+  ERROR_INCORRECT_ENVIRON = "Error: please select to apply develop, test, or stage tag\
```

Comment:
Again, feels a bit too tied to those actual environment names. More generalisation could be desired in the future.